src/gui/filedetails/sharemodel.cpp

```diff
@@ -867,6 +961,42 @@ void ShareModel::toggleShareNoteToRecipientFromQml(const QVariant &share, const
     toggleShareNoteToRecipient(ptr, enable);
 }
 
+void ShareModel::changePermissionMode(const QVariant &share, const SharePermissionsMode permissionMode)
```

Comment:
I believe the naming here should be consistent with the others

```suggestion
void ShareModel::changePermissionModeFromQml(const QVariant &share, const SharePermissionsMode permissionMode)
```

Reply:
@alex ping